src/commands/account/export_account/mod.rs

```diff
@@ -0,0 +1,236 @@
+
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(input_context = crate::GlobalContext)]
+#[interactive_clap(output_context = ExportAccountContext)]
+pub struct ExportAccount {
+    #[interactive_clap(skip_default_input_arg)]
+    /// Which account ID should be exported?
+    account_id: crate::types::account_id::AccountId,
+    #[interactive_clap(subcommand)]
+    export_account_actions: ExportAccountActions,
+}
+
+#[derive(Debug, Clone)]
+pub struct ExportAccountContext {
+    global_context: crate::GlobalContext,
+    account_id: near_primitives::types::AccountId,
+}
+
+impl ExportAccountContext {
+    pub fn from_previous_context(
+        previous_context: crate::GlobalContext,
+        scope: &<ExportAccount as interactive_clap::ToInteractiveClapContextScope>::InteractiveClapContextScope,
+    ) -> color_eyre::eyre::Result<Self> {
+        Ok(Self {
+            global_context: previous_context,
+            account_id: scope.account_id.clone().into(),
+        })
+    }
+}
+
+impl ExportAccount {
+    pub fn input_account_id(
+        context: &crate::GlobalContext,
+    ) -> color_eyre::eyre::Result<Option<crate::types::account_id::AccountId>> {
+        crate::common::input_signer_account_id_from_used_account_list(
+            &context.config.credentials_home_dir,
+            "Which account ID should be exported?",
+        )
+    }
+}
+
+#[derive(Debug, EnumDiscriminants, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(context = ExportAccountContext)]
+#[strum_discriminants(derive(EnumMessage, EnumIter))]
+/// How would you like to export the account?
+pub enum ExportAccountActions {
+    #[strum_discriminants(strum(
+        message = "using-web-wallet          - Export existing account using NEAR Wallet"
+    ))]
+    /// Export existing account using NEAR Wallet
+    UsingWebWallet(self::using_web_wallet::ExportAccountFromWebWallet),
+    #[strum_discriminants(strum(
+        message = "using-seed-phrase         - Export existing account using a seed phrase"
+    ))]
+    /// Export existing account using a seed phrase
+    UsingSeedPhrase(self::using_seed_phrase::ExportAccountFromSeedPhrase),
+    #[strum_discriminants(strum(
+        message = "using-private-key         - Export existing account using a private key"
+    ))]
+    /// Export existing account using a private key
+    UsingPrivateKey(self::using_private_key::ExportAccountFromPrivateKey),
+}
+
+pub fn get_account_key_pair_from_keychain(
+    network_config: &crate::config::NetworkConfig,
+    account_id: &near_primitives::types::AccountId,
+) -> color_eyre::eyre::Result<crate::transaction_signature_options::AccountKeyPair> {
+    let password = get_password_from_keychain(network_config, account_id)?;
+    let account_key_pair = serde_json::from_str(&password);
+    if let Ok(key_pair) = account_key_pair {
+        return Ok(key_pair);
+    }
+    Err(color_eyre::eyre::Report::msg("Error reading data"))
```

Comment:
I believe the following code should work:

```suggestion
    account_key_pair.wrap_err("Error reading data")
```